pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -223,17 +223,17 @@ in (buildEnv {
     sed "1s|$| -I $out/share/texmf/scripts/texlive|" -i "$out/bin/fmtutil"
     ln -sf fmtutil "$out/bin/mktexfmt"
 
-    perl `type -P mktexlsr.pl` ./share/texmf
-    ${bin.texlinks} "$out/bin" && wrapBin
+    perl `type -P mktexlsr.pl` --sort ./share/texmf
+    ${bin.texlinks}/bin/texlinks.sh "$out/bin" && wrapBin
     perl `type -P fmtutil.pl` --sys --all | grep '^fmtutil' # too verbose
-    #${bin.texlinks} "$out/bin" && wrapBin # do we need to regenerate format links?
+    #${bin.texlinks}/bin/texlinks.sh "$out/bin" && wrapBin # do we need to regenerate format links?
 
     # Disable unavailable map files
     echo y | perl `type -P updmap.pl` --sys --syncwithtrees --force
     # Regenerate the map files (this is optional)
     perl `type -P updmap.pl` --sys --force
 
-    perl `type -P mktexlsr.pl` ./share/texmf-* # to make sure
+    perl `type -P mktexlsr.pl` --sort ./share/texmf-* # to make sure
```

Comment:
Cc #102917 – this is a partial fix towards making the combined schemes reproducible.

Reply:
Right, I think several people, including you, had figured out this part, but it was never merged